File: app/src/cs/emailReview.cs
using System;
using System.IO;
using Newtonsoft.Json.Linq;
using Outlook = Microsoft.Office.Interop.Outlook;

class Program
{
    static void Main(string[] args)
    {
        try
        {
            // === Load JSON ===
            JObject json = Util.ReadJsonFromStdin();

            string clientEmail = (string)json["form"]["clientEmail"];
            string clientLanguage = (string)json["form"]["clientLanguage"];

            // === Load the appropriate HTML template ===
            string lang = clientLanguage == "Français" ? "fr" : "en";

            string templatePath = Path.Combine(Util.TemplateDir, lang, "Review.html");

            string htmlBody = File.ReadAllText(templatePath);

            // === Start Outlook and create mail using Interop ===
            var outlookApp = new Microsoft.Office.Interop.Outlook.Application();
            var mail = (Microsoft.Office.Interop.Outlook.MailItem)outlookApp.CreateItem(Microsoft.Office.Interop.Outlook.OlItemType.olMailItem);

            mail.To = clientEmail;
            mail.Subject = clientLanguage == "English"
                ? "Google Review - Allen Madelin"
                : "Commentaires Google - Allen Madelin";
            mail.HTMLBody = htmlBody;
            mail.Display();
            WindowFocus.ShowWithFocus(mail.GetInspector);
        }
        catch (Exception ex)
        {
            Console.WriteLine("Error: " + ex.Message);
        }
    }
}

File: app/src/emailReview.py
from office_utils import *
from parse_json import *

def process_email_review(data):
    try:
        form, _, _ = split_data(data)

        client_email = form["clientEmail"]
        client_language = get_language(data)
        lang = "fr" if client_language.startswith("fr") else "en"

        template_path = os.path.join(TEMPLATES_DIR, lang, "Review.html")
        with open(template_path, "r", encoding="utf-8") as f:
            html_body = f.read()

        outlook = COM.Dispatch("Outlook.Application")
        mail = outlook.CreateItem(0)
        mail.To = client_email
        mail.Subject = "Commentaires Google - Allen Madelin" if lang == "fr" else "Google Review - Allen Madelin"
        mail.HTMLBody = html_body
        mail.Display()

        focus_office_window(mail)
        
        # Return success result
        return {
            "status": "success",
            "message": "Review email draft opened in Outlook.",
            "recipient": client_email,
            "language": lang
        }

    except Exception as e:
        alert_error(f"Error: {e}")
        raise

# Backward compatibility
def main():
    try:
        data = read_json()
        result = process_email_review(data)
        if result.get("error"):
            print(f"Error: {result['error']}")
        else:
            print(result.get("message", "Email review processed"))
    except Exception as e:
        print(f"Error: {str(e)}")

if __name__ == "__main__":
    main()